src/ip.jl

```diff
@@ -55,3 +55,7 @@ function decode_ipv4(data::DenseVector{UInt8})
             p + h.header_length,
             (Int(h.total_length - h.header_length),)))
 end
+
+function ismulticast(ip::IPAddr)
```

Comment:
comparison will fail for IPv6 addresses
```suggestion
function ismulticast(ip::IPv4)
```